Sources/PaymentsCore/AnalyticsEventRequest.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+struct AnalyticsEventRequest: APIRequest {
+    
+    init(payload: AnalyticsPayload) throws {
```

Comment:
I think in other files (and on Braintree) we have the properties before the init. Should we update this to match the other file formats?

Reply:
Good catch! The properties are part of the `APIRequest` protocol implementation, but the `init` is not - so we moved the `init` above the ` // MARK: - APIRequest` to clarify that everything below is part of that protocol. We do this in a few other files too ([example](https://github.com/paypal/iOS-SDK/blob/main/Sources/Card/ConfirmPaymentSourceRequest.swift)).

But I agree it is confusing. The question would be - do we prefer all properties are together, or all protocol stuff is together? I would say the later - but what do you think?

Reply:
Ah, gotcha - I agree I think keeping the protocol stuff together makes sense. Maybe updating protocol conformance marks to `// MARK: - APIRequest conformance` would make this clearer? This is what we've done in Braintree ([example](https://github.com/braintree/braintree_ios/blob/next-major-version/Sources/BraintreeCoreSwift/BTHTTP.swift#L518)) and I think makes it a bit clearer.

Reply:
I like that - especially the name of `APIRequest` isn't very protocol-y (another potential improvement to our networking layer).